### PR TITLE
repositories: add flussence overlay

### PR DESCRIPTION
**Pull request by flussence**
I hope I got this right...

### CODE REVIEW

File: files/overlays/repositories.xml [hunk 1]
```diff
@@ -1596,6 +1596,17 @@ FIN
     </owner>
     <source type="git">https://gitlab.com/Flow/flow-s-ebuilds.git</source>
   </repo>
+  <repo quality="experimental" status="unofficial">
+    <name>flussence</name>
+    <description lang="en">Ant P.'s personal overlay</description>
+    <homepage>https://gitlab.com/flussence/overlay</homepage>
+    <owner type="person">
+      <email>ant+gentoo.bugs@flussence.eu</email>
```
Comment (mgorny) [inline files/overlays/repositories.xml]: This needs to be registered on Gentoo Bugzilla, so people can file bugs.

Comment (flussence) [reply files/overlays/repositories.xml]: [I've been on there for a few years](https://bugs.gentoo.org/buglist.cgi?email1=ant%2Bgentoo.bugs%40flussence.eu&emailreporter1=1&emailtype1=equals&list_id=3823002&query_format=advanced), is registering something different than just having an account?

Comment (mgorny) [reply files/overlays/repositories.xml]: Ok, let me check further. Maybe the script is escaping wrong.

**Comment by flussence** [comment]
Thanks, and sorry for the hassle!

**Pull request closed without merging**